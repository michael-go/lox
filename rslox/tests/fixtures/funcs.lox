fun areWeHavingItYet(suffix) {
  print "Yes we are! " + suffix;
}

print areWeHavingItYet;

areWeHavingItYet("woohoo");