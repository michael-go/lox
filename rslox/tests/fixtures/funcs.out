# exit code: 0
# stdout:
<fn areWeHavingItYet>
Yes we are! foo bar

# stderr:

